docs/products/mysql/concepts/high-availability-mysql.md

```diff
@@ -0,0 +1,132 @@
+---
+title: High availability of Aiven for MySQL®
+sidebar_label: High availability
+---
+
+Aiven for MySQL® is available on a variety of plans, offering different levels of high availability. The selected plan defines the features available.
+
+| Plan         | High availability features                                      | Backup history | Uptime  |
```

Comment:
@alexole @divdby0 @jfg-aiven Could you please have a look at the `Uptime` quotas in the table. They are totally random, just added those we can get the correct values in the review. Do we have such data or shall I remove `Uptime` from the table?

Reply:
I will let product answer this.